cdap-ui/app/features/mapreduce/templates/tabs/runs/run-detail.html

```diff
@@ -19,7 +19,7 @@
         </thead>
         <tbody>
           <td ng-class="{'txt-failed': info.state === 'FAILED', 'txt-running': info.state === 'RUNNING'}"> {{info.state || 'Not Available'}} </td>
-          <td> {{info.startTime || 'Not Available' | date: 'medium' }} </td>
+          <td> {{info.startTime || 'Not Available' | amDateFormat: 'MM/DD/YY h:mm:ss a' }} </td>
```

Comment:
Why is this one different with 'Not Available'?

Also, what happens if you push that string through amDateFormat?